worker/src/lib/managers/AlertManager.ts

```diff
@@ -438,6 +437,16 @@ export class AlertManager {
           } ${await res.text()}`
         );
       }
+
+      if (Object.keys(data || {}).includes("ok")) {
```

Comment:
this is pretty strange. Please use hasOwnedProperty or the `in` operator

"ok" in data